src/CameraControls.ts

```diff
@@ -843,6 +845,8 @@ export class CameraControls extends EventDispatcher {
 
 			if ( ! this._enabled ) return;
 
+			this._resetFocalOffsets();
```

Comment:
I think mouse drag movement does not affect focalOffset. so, can we remove this?

Reply:
The idea is to call this whenever any input starts. Otherwise, we may interrupt transitions or animations. Is there a better place to put this?